checksit/cvs.py

```diff
@@ -25,26 +25,53 @@ def _load(self, vocab_id):
         vocab_file = os.path.join(vocabs_dir, f"{vocab_id}.json")
         self._vocabs[vocab_id] = json.load(open(vocab_file))
 
+    def _load_from_url_ncas(self, vocab_id_url):
+        vocab_id_url_base = vocab_id_url.split("/__latest__")[0]
+        vocab_id_url_base = vocab_id_url_base.replace(
+            "raw.githubusercontent.com", "github.com"
+        )
+        latest_version = requests.get(
+            f"{vocab_id_url_base}/releases/latest"
+        ).url.split("/")[-1]
+        vocab_id_url = vocab_id_url.replace("__latest__", latest_version)
+        res = requests.get(vocab_id_url.replace("__URL__", "https://"))
+        if res.status_code == 200:
```

Comment:
Could also check the status of the previous `requests.get` check on line 33?

Reply:
If the lines above are within an if statement (see my comment https://github.com/cedadev/checksit/pull/53/files/5d1ef42ba0f060b1c9a787f0f2eec306452b5584#r1884087903) then there's a chance the `requests.get` check on line 33 doesn't get run